core-web/apps/dotcms-ui/src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail-form/dot-apps-configuration-detail-form.component.ts

```diff
@@ -36,12 +37,20 @@ export class DotAppsConfigurationDetailFormComponent implements OnInit, AfterVie
     @Output() valid = new EventEmitter<boolean>();
     myFormGroup: UntypedFormGroup;
 
+    constructor(private dotMessageService: DotMessageService) {}
+
     ngOnInit() {
         const group = {};
+
         this.formFields.forEach((field: DotAppsSecret) => {
+            const fieldValue = !!field.editable
```

Comment:
What about adding this logic to `this.getFieldValue` and `getFieldValueFn`

Reply:
Done.